tests/server.py

```diff
@@ -114,8 +114,9 @@ def process(self):
                     ).read_bytes()
                 except (FileNotFoundError, IsADirectoryError):
                     request.setResponseCode(HTTPStatus.NOT_FOUND)
+                request.setHeader("Content-Type", mimetypes.guess_type(uri)[0])
+                request.setHeader("Cache-Control", "no-cache, no-store")
```

Comment:
Needs to be passed as bytes array. :/
```suggestion
                request.setHeader(b"Content-Type", mimetypes.guess_type(uri)[0])
                request.setHeader(b"Cache-Control", "no-cache, no-store")
```